src/request/types/index.ts

```diff
@@ -0,0 +1,43 @@
+import { RpcSuccessResponse } from 'src/types';
+import { GetAddresses, GetInfo, SendTransfer, SignMessage, SignPsbt } from './btcMethods';
+import {
+  StxCallContract,
+  StxDeployContract,
+  StxGetAccounts,
+  StxGetAddresses,
+  StxSignStructuredMessage,
+  StxSignStxMessage,
+  StxSignTransaction,
+  StxTransferStx,
+} from './stxMethods';
+
+export interface StxRequests {
+  stx_callContract: StxCallContract;
+  stx_deployContract: StxDeployContract;
+  stx_getAccounts: StxGetAccounts;
+  stx_getAddresses: StxGetAddresses;
+  stx_signMessage: StxSignStxMessage;
+  stx_signStructuredMessage: StxSignStructuredMessage;
+  stx_signTransaction: StxSignTransaction;
+  stx_transferStx: StxTransferStx;
+}
+
+export type StxRequestMethod = keyof StxRequests;
+
+export interface BtcRequests {
+  getInfo: GetInfo;
+  getAddresses: GetAddresses;
+  signMessage: SignMessage;
+  sendTransfer: SendTransfer;
+  signPsbt: SignPsbt;
+}
+
+export type BtcRequestMethod = keyof BtcRequests;
```

Comment:
is `BtcRequestMethod` or `StxRequestMethod` used anywhere? I see mostly `keyof Requests`

Reply:
it's used on the extension